IntroToCSharp/Pages/Activities/SettingBuilder.razor

```diff
@@ -0,0 +1,150 @@
+@inherits CaptainCoder.Page
+@page "/activities/setting-builder"
+
+<Section Title="Setting Builder">
```

Comment:
First section of each page should be called "Introduction" and include an "Adventure / AdventureDialog". You should put  your Avatar here and introduce yourself. Or, maybe add in a character who is a "City Planner"?

Reply:
They might be a character saying, "It sure is tough to plan a city! I can never remember all of the things that are involved. It would be so helpful if you could create a program to help me remember the steps."